src/main/resources/filing/templates/small-full-accounts.html

```diff
@@ -0,0 +1,1503 @@
+{{"<?xml version=\"1.0\" encoding=\"UTF-8\"?>" | unescaped}}
+
+<html xmlns:ixt2="http://www.xbrl.org/inlineXBRL/transformation/2011-07-31" xmlns="http://www.w3.org/1999/xhtml" xmlns:core="http://xbrl.frc.org.uk/fr/2014-09-01/core" xmlns:xbrldi="http://xbrl.org/2006/xbrldi" xmlns:curr="http://xbrl.frc.org.uk/cd/2014-09-01/currencies" xmlns:countries="http://xbrl.frc.org.uk/cd/2014-09-01/countries" xmlns:uk-bus="http://xbrl.frc.org.uk/cd/2014-09-01/business" xmlns:ix="http://www.xbrl.org/2013/inlineXBRL" xmlns:fn="http://www.w3.org/2005/xpath-functions" xmlns:xbrli="http://www.xbrl.org/2003/instance" xmlns:direp="http://xbrl.frc.org.uk/reports/2014-09-01/direp" xmlns:xlink="http://www.w3.org/1999/xlink" xml:lang="en" xmlns:link="http://www.xbrl.org/2003/linkbase" xmlns:iso4217="http://www.xbrl.org/2003/iso4217">
+  {{$company := .small_full_account.company}}
+  {{$companyName := $company.company_name}}
+  {{$companyNumber := $company.company_number}}
+  <head>
+    <meta content="application/xhtml+xml; charset=UTF-8" http-equiv="content-type" />
+    <title>
+      {{$companyName}}
+    </title>
+    <style type="text/css">
+
+body {
+  font-size: 12px;
+}
+
+/*
+ * Addresses `h1` font sizes within `section` and `article` in Firefox 4+,
+ * Safari 5, and Chrome.
+ */
+h1 {
+    font-size: 2em;
+}
+
+/*
+ * Addresses style set to `bolder` in Firefox 4+, Safari 5, and Chrome.
```

Comment:
Do we need these comments? I assume they come from the abridged template

Reply:
Apologies, didn't realise they corresponded with the styling below. Ignore my comment!